Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -581,6 +584,50 @@ public void TestEnableLoginRetryOn404()
             }
         }
 
+
+        [Test]
+        public void TestNonRetryableHttpExceptionThrowsError()
```

Comment:
it looks like an unit test not an integration test. You could move it to unit tests.

Reply:
Done